tasks.c

```diff
@@ -7568,7 +7568,7 @@ TickType_t uxTaskResetEventItemValue( void )
             {
                 if( xClearCountOnExit != pdFALSE )
                 {
-                    pxCurrentTCB->ulNotifiedValue[ uxIndexToWaitOn ] = 0UL;
+                    pxCurrentTCB->ulNotifiedValue[ uxIndexToWaitOn ] = ( uint32_t ) 0UL;
```

Comment:
Why not TickType_t here?

Reply:
The ulNotifiedValue is of uint32_t type.
https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/37678b06568fda7aaff090b77e8e34d6ab29ede4/tasks.c#L413